Add data cards for public datasets (#1)
audeering/datasets: README.rst

========
datasets
========

Overview_ of publicly available audb_ datasets.
For each dataset
a standardized documentation is provided

> in the form of single data card.

Reviewer: ```suggestion
Overview_ of public datasets
available with audb_.
Each dataset is 
summarized in a data card.
```

> You can easily load all audio files
> and annotations associated with a dataset
> by a single command.
> For example, to load version 1.3.0 of emodb_
> run

Reviewer: ```suggestion
Use the :func:`audb.load`
command to load the audio files
and annotations of a dataset.
The following example
loads version 1.3.0 of emodb_:
```

.. code-block:: python

    db = audb.load('emodb', version='1.3.0')

> compare the `audb quickstart guide`_.

Reviewer: ```suggestion
See also the `audb quickstart guide`_.
```

Reviewer: But this is really my last comment :)

Author: Hehe, doesn't hurt to make this as good as possible ;)
